_includes/code/howto/manage-data.cross-refs.ts

```diff
@@ -78,19 +83,24 @@ try {
 }
 
 // ObjectWithCrossRef
-response = await client.data
-  .creator()
-  .withClassName('JeopardyQuestion')
-  .withId('f7344d30-7fe4-54dd-a233-fcccd4379d5c')
-  .withProperties({
-    'question': 'What tooling helps make Weaviate scalable?',
-    'answer': 'Sharding, multi-tenancy, and replication',
-    'hasCategory': [{  // Specify one or more cross-references
-      'beacon': 'weaviate://localhost/583876f3-e293-5b5b-9839-03f455f14575',
-    }],
-  })
-  .do();
-console.log(JSON.stringify(response, null, 2));
+const myCollection = client.collections.get('WineReviewNV')
+const dataObject = {
+  "title": "A delicious Riesling",
+  "review_body": "This wine is a delicious Riesling which pairs well with seafood.",
+  "country": "Germany",
+}
+
+const uuid = await myCollection.data.insert({
```

Comment:
What do you think about generally making these snippets runnable?

Like:

```ts
import { generateUuid5 } from 'weaviate-client';

const catUuid = '583876f3-e293-5b5b-9839-03f455f14575';
const catProperty = 'hasCategory';
const objUuid = generateUuid5(dataObject.title);

const uuid = await myCollection.data.insert({
  properties: dataObject,
  id: objUuid, // A UUID for the object
  // highlight-start
  references: {
    catProperty: catUuid
  }
  // highlight-end
})

Reply:
that's doable.  i haven't actually worked in the cross-ref examples in their entirety i added this because there's a section under updating data that referenced it ;) 

i put the change in.

Reply:
Yeah no worries. That comment was more generally about not having placeholder strings - let's talk more when we catch up. It's fine for now.